versions.tf

```
@@ -19,11 +19,11 @@ terraform {
   required_providers {
     google = {
       source  = "hashicorp/google"
-      version = ">= 4.0.0, < 5.0"
+      version = ">= 4.0.0, < 6.0"
```

nit: let's use just `6` so in the future the [automated version widen](https://github.com/GoogleCloudPlatform/terraform-google-secret-manager/pull/32) can be used 

```suggestion
      version = ">= 4.0.0, < 6"
```